src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -38,21 +43,24 @@
     box-shadow: $focus-glow-invalid;
 
     &::-ms-check {
-      border-color: var(--hxCheckbox-invalid-bordercolor, $red-status-900);
+      border-color: $red-status-500;
+      border-color: var(--hxCheckbox-invalid-bordercolor, $red-status-500);
```

Comment:
similar setup here. Let me know if this is correct, and I'm just wrong in my prior comment in first file.

Reply:
That's right.  This is our pattern to support IE11 fallbacks.